integration/spark2/src/main/scala/org/apache/spark/sql/execution/command/carbonTableSchema.scala

```diff
@@ -320,7 +322,143 @@ case class AlterTableSplitPartitionCommand(splitPartitionModel: AlterTableSplitP
   }
 }
 
-case class CreateTable(cm: TableModel, createDSTable: Boolean = true) extends RunnableCommand
+case class AlterTableDropPartition(alterTableDropPartitionModel: AlterTableDropPartitionModel)
+  extends RunnableCommand with DataProcessCommand with SchemaProcessCommand {
+  val LOGGER = LogServiceFactory.getLogService(this.getClass.getName)
```

Comment:
why to add so many attribute?

Reply:
The variables could only get from sparkSession and both processSchema and processData will use those variables.